apps/web/src/server/lib/logger.js

```diff
@@ -0,0 +1,35 @@
+import config from '@pins/web/environment/config.js';
+import path from 'node:path';
+import pino from 'pino';
+
+/** @type {import('pino').TransportTargetOptions[]} */
+const targets = [];
+
+if (config.isProd) {
+	targets.push({
```

Comment:
I think we need the pretty version here also, but maybe with a different level. Just to make sure the logstream has the info we need when we look at it.

Reply:
so you want to pretty-print before it's written into the log file? I'd have thought it's better to save them in a one-log-per-line for a log file? (bear in mind they can still be pretty printed from the log file if needed)

Reply:
I would say debug for file and info for console. But I think both should be configured separately.

Reply:
Good point, I wonder if we should replace morgan with https://github.com/pinojs/pino-http to be consistent with the tool and pretty that one to console only for now.

Reply:
Have now updated the PR with a change to transport construction. Regarding pino-http, I'll try it.